libensemble/tests/scaling_tests/balsam_forces/readme.md

```diff
@@ -0,0 +1,61 @@
+## Running test run_libe_forces_balsam.py
+
+Naive Electrostatics Code Test
+
+This is a synthetic, highly configurable simulation function. Its primary use
+is to test libEnsemble's capability to submit application instances via the Balsam service,
+including to separate machines from libEnsemble's processes.
+
+### Forces Mini-App
+
+A system of charged particles is initialized and simulated over a number of time-steps.
+
+Particles' position and charge are initiated using a random stream.
+Particles are replicated on all ranks.
+**Each rank** computes forces for a subset of particles (`O(N^2)` operations).
+Particle force arrays are `allreduced` across ranks.
+Particles are moved (replicated on each rank).
+Total energy is appended to the forces.stat file.
+
+To run forces as a standalone executable on `N` procs:
+
+    mpirun -np N ./forces.x <NUM_PARTICLES> <NUM_TIMESTEPS> <SEED>
+
+This application will need to be compiled on the remote machine where the sim_f will run.
+See below.
+
+### Running with libEnsemble.
+
+On the remote machine:
+
+    git clone https://github.com/argonne-lcf/balsam.git
+    cd balsam; pip install -e .
+    cd ..; balsam site init ./my-site
+
```

Comment:
I would add `balsam login` before `balsam site init ./my-site`. As that line needs to be run afterwards.

Reply:
And then you need to switch to the site directory and run:

`balsam site start`